CONTRIBUTING.md

```diff
@@ -131,8 +131,10 @@ Some common make targets are:
   & everything builds in the correct manner.
 - `make check-stage1-std NO_REBUILD=1` - test the standard library without
   rebuilding the entire compiler
-- `make check TESTNAME=<path-to-test-file>.rs` - Run a single test file
-- `make check-stage1-rpass TESTNAME=<path-to-test-file>.rs` - Run a single
+- `make check TESTNAME=<name-of-test-to-run>` - Run a single test file
+  - `TESTNAME` should be the fully qualified name of the test function, e.g.
```

Comment:
Right, thanks Alex - I've been trying to find how this is matched with not much luck. The reason I wanted to make it a little clearer was when I tried with the file path as suggested I couldn't get it to work - e.g. I used `TESTNAME=src/libstd/collections/hash/map.rs` to try to run the tests there.

I'd like to try to find how this is implemented, so I can update the docs if it is just pattern matching as you say.